src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl

```diff
@@ -1712,25 +1712,109 @@
       },
       "TransactionInfo": {
         "description": "Transaction data returned when querying for an existing transaction",
-        "allOf": [
-          {
-            "$ref": "#/components/schemas/NewTransactionResponse"
+        "type": "object",
```

Comment:
revert those modifications: app io v2 version uses eCommerce v1 for GET transaction that will not be modified (modification will go to the v2 eCommerce GET transaction

Reply:
done here 355365b0040f94471e6d70c9199b909bd64f02b0